lisp/mastodon-http.el

```diff
@@ -100,7 +100,7 @@ Pass response buffer to CALLBACK function."
 (defun mastodon-http--get-json (url)
   "Make GET request to URL. Return JSON response vector."
   (let ((json-vector
-         (with-current-bufer (mastodon-http--get url)
+         (with-current-buffer (mastodon-http--get url)
```

Comment:
Why not squash this commit with the previous one to keep the history sane and readable?